src/test/java/seedu/fast/model/ModelManagerTest.java

```diff
@@ -37,14 +37,14 @@ public void setUserPrefs_nullUserPrefs_throwsNullPointerException() {
     @Test
     public void setUserPrefs_validUserPrefs_copiesUserPrefs() {
         UserPrefs userPrefs = new UserPrefs();
-        userPrefs.setAddressBookFilePath(Paths.get("address/book/file/path"));
+        userPrefs.setFastFilePath(Paths.get("address/book/file/path"));
```

Comment:
I think this Path.get should be changed to "new/fast/file/path"

I think this is just a placeholder file path, and does not actually refer to a real file path